c++/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 project("Cap'n Proto" CXX)
 cmake_minimum_required(VERSION 3.1)
-set(VERSION 0.7-dev)
+set(RELEASE_VERSION 0.7)
```

Comment:
Actually, we want 0.6.0 and 0.6.1 to be different versions, or at least that's how we've done it historically. Our ABI policy is that ABI compatibility in C++ is basically impossible and we don't try to provide it. There's an exception for security patch releases, where we add a fourth number -- e.g. 0.5.3.1 -- and we are very careful to make sure not to touch the ABI.

That said, in practice micro releases (third number) have been pretty ABI-stable, and adding security releases has some annoying side effects. Maybe we should rethink the policy for 0.7?